nautobot/extras/api/serializers.py

```diff
@@ -266,64 +126,8 @@ class ConfigContextSerializer(ValidatedModelSerializer, NotesSerializerMixin):
         default=None,
     )
     owner = serializers.SerializerMethodField(read_only=True)
-    config_context_schema = NestedConfigContextSchemaSerializer(required=False, allow_null=True)
-    locations = SerializedPKRelatedField(
```

Comment:
Can you do a manual test on a `ConfigContext` REST API and make sure all these M2M fields are correctly represented in the response?